api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -1127,13 +1127,14 @@ bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\`
 (id,analysis_id,stratum_1,stratum_2,stratum_3,stratum_4,stratum_5,count_value,source_count_value)
 SELECT 0 as id, 3110 as analysis_id,CAST(sm.concept_id as string) as stratum_1,CAST(o.observation_source_concept_id as string) as stratum_2,
-CAST(o.value_source_concept_id as string) as stratum_3,o.value_as_string as stratum_4,cast(sq.id as string) stratum_5,
+CAST(o.value_source_concept_id as string) as stratum_3,c.concept_name as stratum_4,cast(sq.id as string) stratum_5,
```

Comment:
I was just looking at this coincidentally. What's the issue with value_as_string?

Reply:
prod data seems to have value_as_string many times. (I have verified this only for survey questions).

https://console.cloud.google.com/bigquery?project=all-of-us-workbench-test&angularJsUrl=%2Fbigquery%3Fproject%3Dall-of-us-workbench-test%26j%3Dbquxjob_2f2bfe82_16675780dcb%26page%3Dqueryresults&authuser=3&pli=1&j=bquxjob_280efeb3_1678a1dde17&page=queryresults